### DISCUSSION TITLE
How to position Dropdown/Select content not relevant to the trigger

### DISCUSSION THREAD

**Discussion by majidsajadi** [radix-ui/primitives#1399] [Help] [answered]
Is there any solution to position the content for Select or Dropdown components somewhere else in the page?
For example im developing a PWA which the selects are somehow a bottom sheet. the trigger is in the middle of the page but the content should be fixed at the bottom of the page. 

[Example](https://github.com/gorhom/react-native-bottom-sheet)

**Comment by benoitgrelard** [comment, marked as answer]
Hi @majidsajadi,

This isn't currently possible. But we are tracking the feature request here: #1247
